common/src/main/java/com/sequenceiq/cloudbreak/util/NullUtil.java

```diff
@@ -28,4 +29,11 @@ private NullUtil() {
         }
         return null;
     }
+
+    public static <T, R, U> U getIfNotNull(T firstArgument, R secondArgument, BiFunction<T, R, U> consumer) {
```

Comment:
just cosmetics: the template parameter R is used for the result type, the first 2 parameters' types are T and U respectively